src/lua/call.c

```diff
@@ -617,7 +621,9 @@ static int read_cb(lua_State*L)
   FD_ZERO(&fdset);
   FD_SET(myfileno, &fdset);
   dt_lua_unlock();
+  #if !defined(_WIN32)
```

Comment:
The `read` feature already got disabled under Windows, no need for those if/endif any more.